packages/world/src/World.sol

```diff
@@ -345,18 +364,24 @@ contract World is Store, IWorldCore, IErrors {
    *
    ************************************************************************/
 
+  /**
+   * Allow the World to receive ETH
+   */
+  receive() external payable {}
```

Comment:
There was a warning about a payable fallback function but to receive ether function, and I figured it might be useful to be able to send eth to the world directly (without this you'd need to register a root system with a payable function to allow the World to receive ether)